lib/ViewModels/NowViewingTabViewModel.js

```diff
@@ -118,7 +120,11 @@ NowViewingTabViewModel.prototype.dragStart = function(viewModel, e) {
         e.preventDefault();
     }, false);
 
-    e.dataTransfer.setData('text', 'Dragging a Now Viewing item.');
+    if (defined(e.dataTransfer)) {
+        e.dataTransfer.setData('text', 'Dragging a Now Viewing item.');
+    } else {
+        e.originalEvent.dataTransfer.setData('text', 'Dragging a Now Viewing item.');
```

Comment:
Yeah jQuery inserts itself into all drag events for some reason.  It's good stuff. :)